public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -955,10 +955,13 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
     /**
      *
      * @param panoramaId
+     * @param force: force to change pano, even if walking is disabled
      * @returns {setPano}
      */
-    self.setPano = function (panoramaId) {
-        if (!status.disableWalking) {
+    self.setPano = function (panoramaId, force) {
+        if (force == undefined) force = false;
```

Comment:
Ah I see. I think this is the default parameter implementation. Wasn't aware before this in JS. Still learning :)